core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.html

```diff
@@ -93,7 +106,7 @@
                 (addContent)="addContentlet($event)"
                 data-testId="contentlet-tools" />
             <dot-ema-page-dropzone
-                *ngIf="!!rows.length && !!dragItem && !currentDevice"
```

Comment:
I think that this `dragItem` needs to be here but we are loosing it from a bad merge, I think is related with this https://github.com/dotCMS/core/pull/27290

Reply:
added